src/Nethermind/Nethermind.TxPool/ITxStorage.cs

```diff
@@ -15,4 +15,7 @@ public interface ITxStorage
     IEnumerable<LightTransaction> GetAll();
     void Add(Transaction transaction);
     void Delete(in ValueHash256 hash, in UInt256 timestamp);
+    bool TryGetBlobTransactionsFromBlock(long blockNumber, out Transaction[]? blockBlobTransactions);
+    void AddBlobTransactionsFromBlock(long blockNumber, IList<Transaction> blockBlobTransactions);
+    void DeleteBlobTransactionsFromBlock(long blockNumber);
```

Comment:
Maybe we should extract it to `IBlobTxStorage` interface? Even if same class would implement both (maybe it should be separate class?)

Reply:
We are storing only blob txs for now, doesn't look like we will need to store other types in the future, but if we will - no problem with separating it. I created `IBlobTxStorage` extending `ITxStorage` and moved blob-specific methods